lib/cmd/fh3/credential/create.js

```diff
@@ -0,0 +1,93 @@
+/* globals i18n */
+var fhreq = require('../../../utils/request.js');
+var fs = require('fs');
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('Create a credential bundle'),
+  'examples' :
+  [{
+    cmd : 'fhc credential create --name=<name> --type=<type> --platform=<platform> --env=<env> --privatekey=<privatekey> --certificate=<certificate>',
+    desc : "Create a credential bundle' for Android",
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComma)